pymilvus/client/grpc_handler.py

```diff
@@ -142,7 +143,11 @@ def _setup_grpc_channel(self):
                     with open(self._ca_pem_path, 'rb') as f:
                         root_certificates = f.read()
                     creds = grpc.ssl_channel_credentials(root_certificates, private_key, certificate_chain)
-
+                elif self._server_pem_path != "" and self._server_name != "":
```

Comment:
OK, I got that we don't need another parameter to distinguish one-way tls or two-way tls. If `server_pem_path` and `server_name` were passed, then we should use two-way tls.